src/repl.ts

```diff
@@ -309,6 +331,15 @@ export function repl(prompt: string) {
     if (/^:paste/.test(code) && !multilineBuffer) {
       return enterPasteMode()
     }
+    if (/^:load/.test(code) && !multilineBuffer) {
+      let files = (code.match(/\S+/g) || []).slice(1);
```

Comment:
Why permitting multiple load? I wonder why you need to load file at all. repl is a small playground. If you need load multiple files, `tsc --init` and fire up an editor is clearly better.